src/components/textarea/index.md.njk

```diff
@@ -17,7 +17,11 @@ Use the textarea component when you need to let users enter an amount of text th
 
 ## When not to use this component
 
-Don’t use the textarea component if you need to let users enter shorter answers no longer than a single line, such as a phone number or name. In this case, you should use the [text input](../text-input) component.
+Users can find open-ended questions difficult to answer. It might be better to break up one complex question into a series of simple ones for example where users can select from options using [radios](../radios).
```

Comment:
```suggestion
Users can find open-ended questions difficult to answer. It might be better to break up one complex question into a series of simple ones, for example where users can select from options using [radios](../radios).
```

Reply:
Just one small change from me, otherwise good to go.